premis_event_service/models.py

```diff
@@ -40,7 +40,7 @@ def __str__(self):
         return self.agent_name
 
     def get_absolute_url(self):
-        return reverse('agent-detail', args=[self.agent_identifier])
+        return reverse('agent-detail', args=[str(self.agent_identifier)])
```

Comment:
I don't think this is actually needed. I ran some tests and `self.agent_identifier` is already a string, it doesn't need to be converted.